src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx

```diff
@@ -39,11 +37,7 @@ const VenueWizardCreate: React.FC<VenueWizardCreateProps> = ({
     );
   }
 
-  return (
-    <AdminRestricted>
-      <Details previous={previous} dispatch={dispatch} data={state} />
-    </AdminRestricted>
-  );
```

Comment:
`<AdminRestricted/>` was deleted due to there are nested components
<img width="1438" alt="Screenshot 2021-09-23 at 17 03 48" src="https://user-images.githubusercontent.com/26366773/134636513-ca1ed2c1-2a9c-4d1b-bebf-843c012e9625.png">

Reply:
`AdminRestricted`'s primary role is not visual. Styles can be updated without losing the restriction provided by it.